src/lib/svg_text_utils.js

```diff
@@ -91,7 +91,7 @@ exports.convertToTspans = function(_context, _callback) {
         }
         _context.text('')
             .style({
-                visibility: 'visible',
+                visibility: 'inherit',
```

Comment:
Feel free to pull this fix wherever you need or redo it elsewhere. I won't worry further about it.